src/opera/compare/template_comparer.py

```diff
@@ -0,0 +1,137 @@
+from .diff import Diff
+from .comparisons import Comparison, ListComparison, MapComparison
+from opera.error import DataError
+
+from os import path
+import filecmp
+
+
+class TemplateContext:
+    def __init__(self, template1, template2, workdir1, workdir2):
+        self.template1 = template1
+        self.template2 = template2
+        self.workdir1 = workdir1
+        self.workdir2 = workdir2
```

Comment:
You use words `old` and `new` in python file with diff opera command. Would it be better to use them here as well instead of 1 and 2?

Reply:
Well we use `new` and `old` in diff because it is clear which one is old (the deployed one) and which one is new, Here the comparison is more generic, we do not care actually about the creation time, just the differences.

Reply:
Aha, this is then ok.